components/popover/index.js

```diff
@@ -156,9 +158,23 @@ class Popover extends Component {
 	}
 
 	setOffset() {
-		const { getAnchorRect = this.getAnchorRect } = this.props;
+		const { getAnchorRect = this.getAnchorRect, expandOnMobile = false } = this.props;
 		const { popover } = this.nodes;
 
+		if ( isMobile() && expandOnMobile ) {
+			popover.style.left = 0;
+			popover.style.top = 0;
+			popover.style.right = 0;
+			popover.style.bottom = 0;
+			this.setState( {
+				isMobile: true,
+			} );
+			return;
+		}
+		this.setState( {
+			isMobile: false,
```

Comment:
This will always trigger a re-render, even if `this.state.isMobile` is already `false`.